docs/src/pages/customization/transitions/transitions.md

```diff
@@ -0,0 +1,50 @@
+# Transitions
+
+<p class="description">Use the <code>theme.transitions.create()</code> helper to create consistent transitions on the elements of your UI.</p>
+
+{{"demo": "pages/customization/transitions/TransitionHover.js", "defaultCodeOpen": false}}
```

Comment:
I think that it could be interesting to have a visual example people can directly copy & paste.

```suggestion
```jsx
theme.transitions.create(['background-color', 'transform']);
```

{{"demo": "pages/customization/transitions/TransitionHover.js", "defaultCodeOpen": false}}
```

Reply:
Sry I don’t understand what is the suggested change here. Should I add code snippet after the example?

Reply:
Ah I see the code snippet is added before the example :)